presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -1401,4 +1404,40 @@ public FeaturesConfig setSkipNonApplicableRulesEnabled(boolean skipNonApplicable
         this.skipNonApplicableRulesEnabled = skipNonApplicableRulesEnabled;
         return this;
     }
+
+    public boolean isPrioritizeLargerSpiltsMemoryRevoke()
+    {
+        return prioritizeLargerSpiltsMemoryRevoke;
+    }
+
+    @Config("experimental.prioritize-larger-spilts-memory-revoke")
+    public FeaturesConfig setPrioritizeLargerSpiltsMemoryRevoke(boolean prioritizeLargerSpiltsMemoryRevoke)
+    {
+        this.prioritizeLargerSpiltsMemoryRevoke = prioritizeLargerSpiltsMemoryRevoke;
+        return this;
+    }
+
+    public long getRevocableMemoryLimit()
+    {
+        return revocableMemoryLimit.toBytes();
+    }
+
+    @Config("experimental.revocable-memory-limit")
+    public FeaturesConfig setRevocableMemoryLimit(DataSize revocableMemoryLimit)
+    {
+        this.revocableMemoryLimit = revocableMemoryLimit;
+        return this;
+    }
+
+    public int getSpillPrefetchReadSize()
+    {
+        return spillPrefetchReadSize;
+    }
+
+    @Config("experimental.spill-prefetch-read-size")
```

Comment:
should be read-pages and not read-size...

Reply:
fixed